modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro

```diff
@@ -8,8 +8,6 @@ definition {
 			locator1 = "ObjectPortlet#ENTRY_FIELD_GENERIC",
 			value1 = "${entry}");
 
-		Button.clickSave();
-
 		WaitForSPARefresh();
 
 		WaitForPageLoad();
```

Comment:
Hey,

You could create a new macro for this case and call it separated on testcase. Doesn't make sense to remove only Button.clickSave(), the new macro can contain the three lines (Button.clickSave, WaitForSPARefresh and 
WaitForPageLoad).

So, you can remove the lines on **addCustomFieldEntryViaUI** and call it directly on testcase.

Reply:
If you want, I may explain it better to you.